### PR TITLE
Add members 2024-03-09

### LINKED ISSUE #4804: REQUEST: New membership for atosatto
### GitHub Username

atosatto

### Organization you are requesting membership in

kubernetes

### Requirements

- [X] I have reviewed the [community membership guidelines](https://git.k8s.io/community/community-membership.md)
- [X] I have [enabled 2FA on my GitHub account](https://github.com/settings/security)
- [X] I have subscribed to the [kubernetes dev e-mail list](https://groups.google.com/a/kubernetes.io/group/dev)
- [X] I am actively contributing to 1 or more Kubernetes subprojects
- [X] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [X] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application
- [X] I have verified that my sponsors are from different member companies
- [X] I have verified that my sponsors are a reviewer or an approver in at least one OWNERS file within one of the Kubernetes GitHub organizations (excluding the contributor-playground)
- [ ] **OPTIONAL:** I have taken the [Inclusive Open Source Community Orientation course](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)

### Sponsor 1

soltysh

### Sponsor 2

Huang-Wei

### List of contributions to the Kubernetes project

List of KEPs:
- https://github.com/kubernetes/enhancements/pull/3901

List of PRs:
- https://github.com/kubernetes/kubernetes/pull/98324
- https://github.com/kubernetes/kubernetes/pull/115840
- https://github.com/kubernetes/kubernetes/pull/116687
- https://github.com/kubernetes/kubernetes/pull/119208

### LINKED ISSUE #4809: REQUEST: New membership for ivanvc
### GitHub Username

@ivanvc

### Organization you are requesting membership in

etcd-io

### Requirements

- [X] I have reviewed the [community membership guidelines](https://git.k8s.io/community/community-membership.md)
- [X] I have [enabled 2FA on my GitHub account](https://github.com/settings/security)
- [X] I have subscribed to the [kubernetes dev e-mail list](https://groups.google.com/a/kubernetes.io/group/dev)
- [X] I am actively contributing to 1 or more Kubernetes subprojects
- [X] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [X] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application
- [X] I have verified that my sponsors are from different member companies
- [X] I have verified that my sponsors are a reviewer or an approver in at least one OWNERS file within one of the Kubernetes GitHub organizations (excluding the contributor-playground)
- [ ] **OPTIONAL:** I have taken the [Inclusive Open Source Community Orientation course](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)

### Sponsor 1

@jmhbnz

### Sponsor 2

@ahrtr

### List of contributions to the Kubernetes project

Pull Requests both in progress and merged in @etcd-io repositories:

- etcd: https://github.com/etcd-io/etcd/pulls?q=is%3Apr+author%3Aivanvc
- bbolt: https://github.com/etcd-io/bbolt/pulls?q=is%3Apr+author%3Aivanvc
- Raft: https://github.com/etcd-io/raft/pulls?q=is%3Apr+author%3Aivanvc
- website: https://github.com/etcd-io/website/pulls?q=is%3Apr+author%3Aivanvc

### PR DESCRIPTION
**Pull request by jasonbraganza**
This PR:
- fixes #4804 
- fixes #4809

### CODE REVIEW

File: config/etcd-io/org.yaml [hunk 1]
```diff
@@ -122,15 +132,6 @@ teams:
...
-    members:
-    - jmhbnz
-    - siyuanfoundation
-    - wenjiaswe
-    privacy: closed
-    repos:
-      auger: admin
   members:
     description: ""
     members:
```
Comment (jmhbnz) [inline config/etcd-io/org.yaml:137]: Please also add @ivanvc to the members team so they get the appropriate triage permissions for etcd, thanks.

Comment (jasonbraganza) [reply config/etcd-io/org.yaml:137]: I’d suggest this as as second step once @ivanvc accepts the request and joins the org.

@Priyankasaggu11929, @MadhavJivrajani, please advise

Comment (Priyankasaggu11929) [reply config/etcd-io/org.yaml:137]: @jasonbraganza, yes, adding folks to GH teams is done after org membership is in place.

For now, I think we can add @ivanvc to the requested GH team in a separate commit in this PR and Peribolos should be able to reconcile that.

@jmhbnz, could you please mention the name of the GH teams that @ivanvc needs access to? Thanks!

Comment (jmhbnz) [reply config/etcd-io/org.yaml:137]: Sure thing!  Team is `members`.  Refer diff line 134-135

Comment (jasonbraganza) [reply config/etcd-io/org.yaml:137]: @Priyankasaggu11929, done. Please take a look.

Comment (jmhbnz) [reply config/etcd-io/org.yaml:137]: I'm not seeing `ivanvc` in the members team @jasonbraganza?

Comment (jasonbraganza) [reply config/etcd-io/org.yaml:137]: Well I’ll be … sorry folks! will go have a look see later in the day. apologies!

Comment (jasonbraganza) [reply config/etcd-io/org.yaml:137]: think I licked it. @Priyankasaggu11929, now will you ptal?

**Review by jmhbnz — APPROVED** [review]
/lgtm

**Pull request merged**